src/examples/standalone/langchain/index.ts

```diff
@@ -0,0 +1,160 @@
+import readline from "node:readline";
+import { ChatAnthropic } from "@langchain/anthropic";
+import { AgentExecutor, createStructuredChatAgent } from "langchain/agents";
+import { pull } from "langchain/hub";
+import type { ChatPromptTemplate } from "@langchain/core/prompts";
+import { HumanMessage, AIMessage } from "@langchain/core/messages";
+import { getOnChainTools } from "@goat-sdk/adapter-langchain";
+import { viem } from "@goat-sdk/wallet-viem";
+import { createWalletClient, http } from "viem";
+import { privateKeyToAccount } from "viem/accounts";
+import { bosonProtocolPlugin } from "@bosonprotocol/agentic-commerce";
+import { BOSON_MCP_URL, CHAIN_MAP } from "@common/chains.ts";
+
+// Example test for the Boson MCP Server plugin
+async function testBosonMcpServerPlugin() {
```

Comment:
```suggestion
async function main() {
```

I'll change it in the other examples